docs/src/examples/DropdownExample/DropdownExampleFifth.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { Dropdown, DropdownItemProps } from '../../../../src/components/';
+import * as styles from '../../styles/components-page.scss';
+
+export interface IProps{
+}
+
+export interface IState {
+  active: boolean;
+}
+
+class DropdownExampleFifth extends React.Component<IProps, IState> {
+  constructor(props: IProps) {
+    super(props);
+    this.state = {
+      active: true,
+    };
+    this.toggle = this.toggle.bind(this);
+  }
+
+  private toggle() {
+    this.setState({
+      active: !this.state.active
+    });
+  }
+
+  render() {
+    const items : DropdownItemProps[] = [
+      {
+        children: "Item 1",
+        header: false,
```

Comment:
@shyamk9 No need to add these false flag, remove all false flag from json & consider them as false in dropdown component if you don't receive the respective flag, add it only for true value